pkg/apiserver/logsearch/task.go

```diff
@@ -211,8 +231,7 @@ func (t *Task) SyncRun() {
 		}
 		for _, msg := range res.Messages {
 			line := logMessageToString(msg)
-			// TODO: use unsafe here: string -> []byte
-			_, err := bufWriter.Write([]byte(line))
+			_, err := bufWriter.Write(*(*[]byte)(unsafe.Pointer(&line)))
```

Comment:
```
*(*[]byte)(unsafe.Pointer(&line))
```
wrapper it to a common function

Reply:
I think just using inline converting is more clear semantically